README.txt

```diff
@@ -3,7 +3,7 @@ Contributors: constantcontact, webdevstudios, znowebdev, jmichaelward, ggwicz, r
 Tags: capture, contacts, constant contact, constant contact form, constant contact newsletter, constant contact official, contact forms, email, form, forms, marketing, mobile, newsletter, opt-in, plugin, signup, subscribe, subscription, widget
 Requires at least: 5.2.2
 Tested up to: 5.3.0
```

Comment:
Can we get the tested up to bumped up to 5.4.0, assuming we're testing on that? If not, we probably should. Otherwise everything looks good to me.